.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
How would you feel about renaming the `master` branch to `main` while we're touching this project? Are there other connections (e.g. Jenkins, etc.) that would need to be updated if we did that?

Reply:
I need to check with Stephen/Amthony to see how Jenkins communicates with the repo, and how best to wire Jenkins in with actions.

Reply:
Looks like (from the deploy.sh) that you need to hit the Jenkins webhook (at the bottom of that file). Fwiw, there are GitHub Actions for adding an SSH key too (which I see that deploy.sh also does). Those things that were in secrets in Travis can be put in GA secrets.